AlphaWallet/Wallet/ViewControllers/suggested_keyword.plist

```diff
@@ -0,0 +1,2054 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<array>
+    <string>abandon</string>
```

Comment:
@nimitparekh2020 will review. But a quick feedback. This file doesn't need to be added. The word list is available as `HDWallet.englishWordList`.

Reply:
I will create new function into `HDWallet` class, which returns all `mnemonic` then It fetches into into `ViewController` and I will remove created `.plist`.